webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path")
+
+const { NODE_ENV = "development" } = process.env
+
+module.exports = {
+    mode: NODE_ENV,
+    devtool: "sourcemap",
+    entry: "./src/index.ts",
+    stats: "errors-only",
```

Comment:
If your curious about stats around bundle sizes and what files are being compiled, comment out this line. I set it to `errors-only` so that it's less verbose.